src/usr/local/www/services_ntpd.php

```diff
@@ -70,6 +70,12 @@
 		$input_errors[] = gettext("The supplied value for Maximum Poll Interval is invalid.");
 	}
 
+	for ($i = 0; $i < NUMTIMESERVERS; $i++) {
+		if (isset($pconfig["servselect{$i}"]) && isset($pconfig["servispool{$i}"])) {
```

Comment:
If you look at https://github.com/pfsense/pfsense/blob/master/src/etc/inc/system.inc#L1776 and https://github.com/pfsense/pfsense/blob/master/src/usr/local/www/services_ntpd.php#L282 you can see that the system assumes an address is a pool if it ends with the value in `$auto_pool_suffix` (currently `pool.ntp.org`), so this validity check should also perform the same test.

You will need to move the definition of `$auto_pool_suffix` up higher (or perhaps make it global in globals.inc and change all references to it so it reads from there).

Reply:
ok, fixed